bin/ssh_scan

```diff
@@ -35,8 +35,15 @@ opt_parser = OptionParser.new do |opts|
       exit
     end
     File.open(file).each do |line|
-      line.chomp.split(',').each do |ip|
+      line.chomp.split(',').each do |ips|
+        ip_arr = ips.chomp.split(':')
```

Comment:
Instead of getting things into array you could do

ip, port = socket.chomp.split(':')

and then check if port is nil then put the default 22 port